vault/core_metrics.go

```diff
@@ -69,8 +70,19 @@ func (c *Core) metricsLoop(stopCh chan struct{}) {
 				continue
 			}
 			if c.perfStandby { // already have lock here, do not re-acquire
-				syncCounter(c)
+				err := syncCounters(c)
+				if err != nil {
+					c.logger.Error("writing syncing counters", "err", err)
+				}
 			} else {
+				// Perf standbys will have synced above, but active nodes on a secondary cluster still need to ship
+				// barrier encryption counts
+				if c.ReplicationState().HasState(consts.ReplicationPerformanceSecondary) {
+					err := syncBarrierEncryptionCounter(c)
```

Comment:
From https://app.circleci.com/pipelines/github/hashicorp/vault/15613/workflows/39275698-fbdc-49ea-89de-ea876aa43e4d/jobs/158313 , this function probably needs to be pulled from the ent changes.

Reply:
hmm, yeah, tricky.